compiler/rustc_middle/src/ty/vtable.rs

```diff
@@ -94,6 +95,7 @@ impl<'tcx> TyCtxt<'tcx> {
                 .expect("failed to build vtable representation");
         }
 
+        vtable.update_mutability(Mutability::Not);
```

Comment:
The field is public, you should not have to add a method for this.

Reply:
Indeed! Thanks